### DISCUSSION TITLE
No matching version found for @babel/helper-plugin-utils@^7.22.5.

### DISCUSSION THREAD

**Discussion by Saber-youMianJing** [babel/babel#15693] [Q&A]
npm ERR! notarget No matching version found for @babel/helper-plugin-utils@^7.22.5.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

There were no changes to package.json, and suddenly there was a problem using Angular as the framework

**Comment by nicolo-ribaudo** [comment]
Version 7.22.5 is on npm https://www.npmjs.com/package/@babel/helper-plugin-utils 🤔 

Please see this answer: https://github.com/babel/babel/issues/15654#issuecomment-1564520404
